server/api-util/lineItemHelpers.js

```diff
@@ -20,12 +20,12 @@ const LINE_ITEM_DAY = 'line-item/day';
  * @returns {Money} lineTotal
  */
 exports.calculateTotalPriceFromQuantity = (unitPrice, unitCount) => {
-  const numericPrice = convertMoneyToNumber(unitPrice);
-  const numericTotalPrice = new Decimal(numericPrice).times(unitCount).toNumber();
-  return new Money(
-    convertUnitToSubUnit(numericTotalPrice, unitDivisor(unitPrice.currency)),
-    unitPrice.currency
-  );
+  const amountFromUnitPrice = getAmountAsDecimalJS(unitPrice);
+  const totalPrice = amountFromUnitPrice.times(unitCount);
```

Comment:
I think we need rounding here too as unitCount (which I assume is the same as quantity) can be a non-integer